stronk/models/workout_exercise.py

```diff
@@ -29,15 +29,20 @@ class WorkoutExercise(db.Model):
                             primary_key=True,
                             index=True,
                             nullable=False)
+    superset_exercise_id = db.Column(db.Integer,
+                                      db.ForeignKey('exercise.id'),
+                                      primary_key=False,
```

Comment:
Aren't columns not primary keys by default?

Reply:
True, will fix :)